RDVFSharp/Commands/RoomCreation/RoomJoin.cs

```diff
@@ -41,21 +41,36 @@ public async Task<List<string>> Execute(string characterCalling, IEnumerable<str
 
         public async new void ExecutePrivateCommand(string characterCalling, IEnumerable<string> args)
         {
-            var result = await Execute(characterCalling, args);
-            foreach (var message in result)
+            if (characterCalling == "Mayank" || characterCalling == "Elise Pariat" || characterCalling == "Aelith Blanchette")
```

Comment:
Use "public static string ADMIN_MAYANK" instead of inline strings like that. It'll make it easier to check where our names are in the code later on